addons/t9n/models/project.py

```diff
@@ -23,3 +25,10 @@ class Project(models.Model):
         string="Languages",
         help="The list of languages into which the project can be translated.",
     )
+
+    @api.constrains('src_lang_id', 'target_lang_ids')
+    def _check_source_and_target_languages(self):
+        for record in self:
+            if record.src_lang_id in record.target_lang_ids:
+                raise ValidationError(
+                    message="The source language can not be one of the target languages.")
```

Comment:
You may want to make the error message translatable 😺
```suggestion
                    message=_("The source language can not be one of the target languages."))
```

Reply:
Fixed!